kernel/async/monitor-common/async/Place.cc

```diff
@@ -43,17 +43,12 @@ namespace async {
 
   void Place::processTasks()
   {
-    auto msg = queue.pull();
     while (true) {
-      if (msg != nullptr) {
-        MLOG_DETAIL(mlog::async, this, "run tasklet", msg);
-        msg->run();
-      } else {
-        if (queue.tryRelease()) break;
-        hwthread_pause();
```

Comment:
Waiting here should be wrong because we already know that there is a new message.